src/sagemaker/jumpstart/estimator.py

```diff
@@ -730,10 +730,10 @@ def attach(
             ValueError: if the model ID or version cannot be inferred from the training job.
 
         """
-
+        config_name = None
         if model_id is None:
 
-            model_id, model_version = get_model_id_version_from_training_job(
+            model_id, model_version, config_name = get_model_id_version_from_training_job(
```

Comment:
Can you update the function to be more generic, since config is now returned?

Reply:
Yes I'll update